### PR TITLE
Set up internet warriors

### PR DESCRIPTION
**Pull request by duarte**
Just adds a temporary link for the next doc, Internet Warriors

### CODE REVIEW

File: src/js/lib/sheetNameFromShortId.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export default function sheetNameFromShortId(docsArray, shortUrl) {
     const map = new Map(docsArray);
     const docName = map.get(shortUrl);
-    return docName ? docName : 'gun-nation';
+    return docName ? docName : 'internet-warriors';
```
Comment (akash1810) [inline src/js/lib/sheetNameFromShortId.js]: why this change?

Comment (gidsg) [inline src/js/lib/sheetNameFromShortId.js]: This is a workaround to the fact that now we are relying on `pageId` we don't know this until the day of publication, so we default to the latest doc.
